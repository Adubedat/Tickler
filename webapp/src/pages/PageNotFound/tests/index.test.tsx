import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import { PageNotFound } from "../index";

describe("<PageNotFound />", () => {
  test("renders", () => {
    const wrapper = shallow(<PageNotFound />);

    expect(toJson(wrapper.dive())).toMatchSnapshot();
  });
});