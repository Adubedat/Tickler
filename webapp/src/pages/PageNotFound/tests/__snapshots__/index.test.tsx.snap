// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageNotFound /> renders 1`] = `
<div
  className="sc-bdfBwQ iVkvtj"
>
  <p>
    404 Error Page not found
  </p>
</div>
`;
